src/commands/commit/prompts.js

```diff
@@ -23,40 +25,50 @@ export type Answers = {
   message: string
 }
 
-export default (gitmojis: Array<Gitmoji>): Array<Object> => [
-  {
-    name: 'gitmoji',
-    message: 'Choose a gitmoji:',
-    type: 'autocomplete',
-    source: (answersSoFor: any, input: string) => {
-      return Promise.resolve(
-        filterGitmojis(input, gitmojis).map((gitmoji) => ({
-          name: `${gitmoji.emoji}  - ${gitmoji.description}`,
-          value: gitmoji[configurationVault.getEmojiFormat()]
-        }))
-      )
-    }
-  },
-  ...(configurationVault.getScopePrompt()
-    ? [
-        {
-          name: 'scope',
-          message: 'Enter the scope of current changes:',
-          validate: guard.scope
-        }
-      ]
-    : []),
-  {
-    name: 'title',
-    message: 'Enter the commit title',
-    validate: guard.title,
-    transformer: (input: string) => {
-      return `[${input.length}/${TITLE_MAX_LENGTH_COUNT}]: ${input}`
+export default (gitmojis: Array<Gitmoji>, mode: CommitMode): Array<Object> => {
+  const { title, message } = mode === 'hook'
+    ? getDefaultCommitContent()
+    : { title: undefined, message: undefined }
```

Comment:
I have a question right here, If we are returning `{ title: undefined, message: undefined }` in here, what's the point then of returning it also inside of the `getDefaultCommitContent` function ?

I mean can we find another way to remove the default values for title and message prompts where the mode it's not hook ?

Reply:
I added the `mode` parameter to `getDefaultCommitContent` util so that we can return null values directly in the function when `mode === 'client'`